activesupport/test/deprecation_test.rb

```diff
@@ -130,7 +132,7 @@ def test_several_behaviors
     assert_equal 4, @c.size
   end
 
-  def test_raise_behavior
+  test ":raise behavior" do
```

Comment:
Let's keep all tests in this file consistent by adding another commit changing the syntax to use the declarative.

Reply:
Ok I will keep these the same, and change them in a follow-up commit.

In the past, I have been naming new tests to be consistent with the style of the existing tests.  But my opinion has recently changed.  I feel like new tests should be written with the declarative style so that, over time, the code base naturally drifts towards a majority of declarative tests.

Do you have a strong preference?

Reply:
I'm ok with both. I'd just void creating inconsistencies inside the same file.

Reply:
I have changed the existing tests to the declarative syntax in #46018. :+1: